Interface/DragoonHudWindow.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Numerics;
+using Dalamud.Game.ClientState.Structs.JobGauge;
+using Dalamud.Plugin;
+using ImGuiNET;
+
+namespace DelvUIPlugin.Interface
+{
+    public class DragoonHudWindow : HudWindow
+    {
+        public override uint JobId => 32;
+
+        private new int BarHeight => 13;
+        private new int BarWidth => 254;
+        private new int XOffset => 127;
+        private new int YOffset => 466;
+
+        private readonly int xPadding = 2;
+
+        public DragoonHudWindow(DalamudPluginInterface pluginInterface, PluginConfiguration pluginConfiguration) : base(pluginInterface, pluginConfiguration) { }
+
+        protected override void Draw(bool _)
+        {
+            DrawHealthBar();
+            DrawPrimaryResourceBar();
+            DrawSecondaryResourceBar();
+            DrawTargetBar();
+        }
+
+        protected override void DrawPrimaryResourceBar()
+        {
+            var gauge = PluginInterface.ClientState.JobGauges.Get<DRGGauge>();
+
+            var barWidth = (BarWidth - xPadding * 2) / 2.0f;
+            var barSize = new Vector2(barWidth, BarHeight);
+            var xPos = CenterX - XOffset;
+            var yPos = CenterY + YOffset - 33;
+            var cursorPos = new Vector2(xPos, yPos);
+            var eyeCount = gauge.EyeCount;
+
+            // Chunk 1
+            var drawList = ImGui.GetWindowDrawList();
+            if (eyeCount >= 1)
+            {
+                drawList.AddRectFilled(cursorPos, cursorPos + barSize, 0xFFC2B6FF);
```

Comment:
Is there somewhere that might be better suited for storing these colors? Since it seems like they're meant to be used multiple times might make future maintenance easier if these were constants (and later loaded in via configuration if the functionality is so desired)